src/daemon/rpm-ostreed-automatic.service.in

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=RPM-OSTree Automatic Update
+Documentation=man:rpm-ostree(1) man:rpm-ostreed.conf(5)
+ConditionPathExists=/run/ostree-booted
+
+[Service]
+Type=oneshot
+# give lowest priority possible; copied from dnf-automatic.service, see systemd.exec(5)
+Nice=19
+IOSchedulingClass=2
+IOSchedulingPriority=7
```

Comment:
But none of that is really useful here since we're making a DBus request to the daemon.

Reply:
Oh definitely, I was too hasty on that one.